climada/entity/exposures/litpop/litpop.py

```diff
@@ -0,0 +1,1199 @@
+                built assets such as machinery, equipment, physical structures,
+                and land value for built-up land.
+                (pc is in constant 2014 USD)
+        * 'pop': population count (source: GPW, same as gridded population)
+        * 'gdp': gross-domestic product (Source: World Bank)
+        * 'income_group': gdp multiplied by country's income group+1
+        * 'nfw': non-financial wealth (Source: Credit Suisse, of households only)
+        * 'tw': total wealth (Source: Credit Suisse, of households only)
+        * 'norm': normalized by country
+        * 'none': LitPop per pixel is returned unscaled
+        The default is 'pc'
+    reference_year : int
+        reference year for data extraction
+    total_population : number, optional
+        total population number, only required for fin_mode 'pop'.
+        The default is None.
+
+    Returns
+    -------
+    total_value : float
+    """
+    if fin_mode == 'none':
+        return None
+    if fin_mode == 'pop':
+        return total_population
+    if fin_mode == 'pc':
+        return(u_fin.world_bank_wealth_account(cntry_iso3a, reference_year,
+                                               no_land=True)[1])
+        # here, total_asset_val is Produced Capital "pc"
+        # no_land=True returns value w/o the mark-up of 24% for land value
+    if fin_mode == 'pc_land':
+        return(u_fin.world_bank_wealth_account(cntry_iso3a, reference_year,
+                                               no_land=False)[1])
+        # no_land=False returns pc value incl. the mark-up of 24% for land value
+    if fin_mode == 'norm':
+        return 1
+    # GDP based total values:
+    gdp_value = u_fin.gdp(cntry_iso3a, reference_year)[1]
+    if fin_mode == 'gdp':
+        return gdp_value
+    if fin_mode == 'income_group': # gdp * (income group + 1)
+        return gdp_value*(u_fin.income_group(cntry_iso3a, reference_year)[1]+1)
+    if fin_mode in ('nfw', 'tw'):
+        wealthtogdp_factor = u_fin.wealth2gdp(cntry_iso3a, fin_mode == 'nfw',
+                                              reference_year)[1]
+        if np.isnan(wealthtogdp_factor):
+            LOGGER.warning("Missing wealth-to-gdp factor for country %s.", cntry_iso3a)
+            LOGGER.warning("Using GDP instead as total value.")
+            return gdp_value
+        return gdp_value * wealthtogdp_factor
+    raise ValueError(f"Unsupported fin_mode: {fin_mode}")
+
+def reproject_input_data(data_array_list, meta_list,
+                        i_ref=0,
+                        target_res_arcsec=None,
+                        global_origins=(-180.0, 89.99999999999991),
+                        resampling=rasterio.warp.Resampling.bilinear,
+                        conserve=None):
+    """
+    Reprojects all arrays in data_arrays to a given resolution –
+    all based on the population data grid.
+
+    Parameters
+    ----------
+    data_array_list : list or array of numpy arrays containing numbers
```

Comment:
I can't provide a target_projection at this point, because the target_projection needs to be constructed from the reference meta and the target_resolution to ensure that all LitPop data on a certain target_resolution are aligned, i.e. on the same global grid. This is the reason for this construct. The alignment happens within `climada.util.coordinates.align_raster_data()` which is called by `litpop.reproject_input_data()`. 
See my reply below (https://github.com/CLIMADA-project/climada_python/pull/267#discussion_r700335061) for further elaboration and my suggestion on renaming `i_ref` to `i_align`...

Reply:
ok, thanks for explaining! 
no strong opinions to i_ref or i_align, tough.